frontend/app/admin/src/views/organizations/tags/EditOrganizationTagsView.vue

```diff
@@ -54,13 +101,13 @@ async function addTag() {
         modalDisplayStyle: 'popup',
         components: [
             new ComponentWithProperties(EditOrganizationTagView, {
+                allTags: tags.value,
                 tag,
                 isNew: true,
-                saveHandler: (patch: AutoEncoderPatchType<OrganizationTag>) => {
-                    patch.id = tag.id;
-                    arr.addPatch(patch);
-                    addArrayPatch(arr);
+                saveHandler: (patches: { tag: OrganizationTag; patch: AutoEncoderPatchType<OrganizationTag> }[]) => {
+                    applyPatches(patches, arr);
                 },
+                deleteHandler,
```

Comment:
The delete handler here is not required here because the tag is not yet added - it is new (the delete will fail anyway)